draft-ietf-quic-transport.md

```diff
@@ -1477,35 +1481,270 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
 
 ## Connection Migration {#migration}
 
-QUIC connections are identified by their 64-bit Connection ID.  QUIC's
-consistent connection ID allows connections to survive changes to the client's
-IP and/or port, such as those caused by client or server migrating to a new
-network.  Connection migration allows a client to retain any shared state with a
-connection when they move networks.  This includes state that can be hard to
-recover such as outstanding requests, which might otherwise be lost with no easy
-way to retry them.
+QUIC allows connections to survive changes to endpoint addresses (that is, IP
+address and/or port), such as those caused by a client migrating to a new
+network.  This section describes the protocol for migrating a connection to a
+new client address.
+
+Migrating a connection to a new server address is left for future work. If a
+client receives packets in the connection from a new server address, the client
+MAY discard these packets.
+
+
+### Path Validation {#migrate-validate}
+
+Path validation is used by the migrating endpoint to verify reachability of
+its peer from a new local address. Path validation is also used by the peer to
+verify that the migrating endpoint is able to receive packets sent to that
+address.  That is, that the packets received from the migrating endpoint do
+not carry a spoofed source address.
+
+Path validation MAY be used at any time by either endpoint.  For instance, an
+endpoint might check that a peer is still in possession of its address after a
+period of quiescence.
+
+An endpoint MAY bundle PATH_CHALLENGE and PATH_RESPONSE frames with other
+frames, as appropriate.  For instance, an endpoint may pad a packet carrying a
+PATH_CHALLENGE for PMTU discovery, or an endpoint may bundle a PATH_RESPONSE
+with its own PATH_CHALLENGE.
+
+
+#### Initiation
+
+To initiate path validation, an endpoint sends a PATH_CHALLENGE frame containing
+a payload that is hard to guess on the path to be validated.
+
+An endpoint MAY send additional PATH_CHALLENGE frames to handle packet loss,
+but is subject to the following limit to avoid excessive network load: an
+endpoint SHOULD NOT send a PATH_CHALLENGE more frequently than it would a client
+INITIAL, ensuring that connection migration is no more load on a new path than
```

Comment:
Packet names are title-cased, not all-caps.  Also, I'd drop the colon lead-up.

`...but to avoid excessive network load, an endpoint SHOULD NOT... an Initial packet.  This ensures that....`

Reply:
Fixed.